test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -40,6 +40,7 @@ namespace test {
 
 eth::Network ChainBranch::s_tempBlockchainNetwork = eth::Network::MainNetwork;
 eth::Network TestBlockChain::s_sealEngineNetwork = eth::Network::FrontierTest;
+TestBlockChain::MiningType getMiningType(json_spirit::mObject const& _testObj);
```

Comment:
Better put some empty lines between your constants and function declarations, definitions